doc/orchestrator.rst

```diff
@@ -63,108 +326,207 @@ on that compute host.
-specifying the optional argument ``custom_pinning``, an alternative pinning can be specified
-by sending in a list of CPU ids (e.g [0,2,range(5,8)]). For optimal performance, most users
-will want to also modify the RunSettings for the model to pin their application to cores not
-occupied by the database.
+    import numpy as np
+    from smartredis import Client
+    from smartsim import Experiment
+    from smartsim.log import get_logger
+    import sys
 
-.. warning::
+    exe_ex = sys.executable
+    logger = get_logger("Example Experiment Log")
+    # Initialize the Experiment
+    exp = Experiment("getting-started", launcher="auto")
 
-  Pinning is not supported on MacOS X. Setting ``custom_pinning`` to anything
-  other than ``None`` will raise a warning and the input will be ignored.
+Initialize a Colocated Model
+----------------------------
+In the next stage of the experiment, we
+create and launch a colocated ``Model`` that
+runs the application script with a database
+on the same compute node.
+
+Step 1: Configure
+"""""""""""""""""
+In this experiment, we invoke the Python interpreter to run
+the python script defined in section: The Application Script.
+To configure this into a ``Model``, we use the ``Experiment.create_run_settings()`` function.
+The function returns a ``RunSettings`` object.
+A ``RunSettings`` allows you to configure
+the run settings of a SmartSim entity.
+We initialize a RunSettings object and
+specify the path to the application file,
+`application_script.py`, to the argument
+``exe_args``, and the run command to ``exe``.
 
 .. note::
+  Change the `exe_args` argument to the path of the application script
+  on your file system to run the example.
 
-  Pinning _only_ affects the co-located deployment because both the application and the database
-  are sharing the same compute node. For the clustered deployment, a shard occupies the entirety
-  of the node.
+Use the ``RunSettings`` helper functions to
+configure the the distribution of computational tasks (``RunSettings.set_nodes()``). In this
+example, we specify to SmartSim that we intend the Model to run on a single compute node.
 
-Redis
-=====
+.. code-block:: python
 
-.. _Redis: https://github.com/redis/redis
-.. _RedisAI: https://github.com/RedisAI/RedisAI
+    # Initialize a RunSettings object
+    model_settings = exp.create_run_settings(exe=exe_ex, exe_args="/lus/scratch/richaama/clustered_model.py")
+    # Configure RunSettings object
+    model_settings.set_nodes(1)
 
-The ``Orchestrator`` is built on `Redis`_. Largely, the job of the ``Orchestrator`` is to
-create a Python reference to a Redis deployment so that users can launch, monitor
-and stop a Redis deployment on workstations and HPC systems.
+Step 2: Initialize
+""""""""""""""""""
+Next, create a ``Model`` instance using the ``Experiment.create_model()``.
+Pass the ``model_settings`` object as an argument
+to the ``create_model()`` function and assign to the variable ``model``.
```

Comment:
assign to the variable ``model`` 

->

assign the returned ``Model`` object to the variable ``model``